servers/audio_server.h

```diff
@@ -428,9 +429,9 @@ class AudioServer : public Object {
 	float get_playback_speed_scale() const;
 
 	// Convenience method.
-	void start_playback_stream(Ref<AudioStreamPlayback> p_playback, const StringName &p_bus, Vector<AudioFrame> p_volume_db_vector, float p_start_time = 0, float p_pitch_scale = 1);
+	void start_playback_stream(Ref<AudioStreamPlayback> p_playback, const StringName &p_bus, Vector<AudioFrame> p_volume_db_vector, float p_from_pos = 0, float p_scheduled_start_time = 0, float p_pitch_scale = 1);
```

Comment:
_process() 's delta uses double. Do we want to match?

Reply:
Done.